sdk/go/auto/stack.go

```diff
@@ -142,9 +142,6 @@ type Stack struct {
 // FullyQualifiedStackName returns a stack name formatted with the greatest possible specificity:
 // org/project/stack or user/project/stack
 // Using this format avoids ambiguity in stack identity guards creating or selecting the wrong stack.
-// Note that filestate backends (local file, S3, Azure Blob) do not support stack names in this
```

Comment:
Some users _are_ still on the legacy filestate structure without project support, plus while new filestate backends do support fully qualified names they are limited to the org being set to "organization", that should probably be called out here.

Reply:
Should this comment stay as is but add the "organization" requirement, or remove it and replace it with the "organization" requirement?

Reply:
I'd probably update it to say "legacy filestate backends" for this sentence, and add another sentence probably before it about org having to be "organization" for filestate. But whatever you think is more clear is probably right here, being a user of the system.

Reply:
I've updated the comment. Let me know what you think.

Reply:
Yeh this looks good. Definitely better than before so ✅